device/iot-device-samples/send-serialized-event/src/main/java/samples/com/microsoft/azure/sdk/iot/ContosoAnemometer.java

```diff
@@ -7,7 +7,9 @@
 
 public class ContosoAnemometer {
     public String deviceId;
-    public double windSpeed;
+    public int messageId;
+    public double temperature;
```

Comment:
Can you rename messageId to somehting else so that it doesn't coincide with messageID and confuse the user ?